github/.gitignore

```diff
@@ -0,0 +1,2 @@
+Makefile
```

Comment:
Why `Makefile`?

Reply:
sigh - probably hysterical raisins.  I'll research and/or remove.

Reply:
removed in next commit